<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRESPSet</class>
 <widget class="QDialog" name="DialogRESPSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Gain</string>
   </property>
   <widget class="QRadioButton" name="radioButton1X">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton4X">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2X">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton0_25X">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.25X</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Resp lead</string>
   </property>
   <widget class="QRadioButton" name="radioButtonRA_LL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RA-LL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRA_LA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RA-LA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLA_RL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LA-RL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRL_LL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RL-LL</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxApnea">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>open apnea alarm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditApnea">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>sec</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RESP threshold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditThreshold">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAntidrift">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>open antidrift</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxWaveSpeed">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>6.25mm/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12.5mm/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25mm/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50mm/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100mm/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>full speed</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>wave speed:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogRESPSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogRESPSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
